tools/f18/f18.cc

```diff
@@ -101,6 +101,9 @@ struct DriverOptions {
   bool unparseTypedExprsToPGF90{false};
   std::vector<std::string> pgf90Args;
   const char *prefix{nullptr};
+  bool getDefinition{false};
+  bool getSymbolsSources{false};
+  int sourcePosition[3];
```

Comment:
Can you please document or rename this magic three-`int` encoding?  Maybe you could create a struct for it and pass that struct as an argument below rather than marshaling the three `int`s as individual arguments.

Reply:
I have renamed and commented the variable. I could not find a clean way to add a struct for this.

Reply:
What were the problems that prevented you from defining a `struct`?

Reply:
To add a struct, I have to loop through the 3 arguments to validate them and store them in a temporary int array before putting it into the struct, which seems moot because the result is still an array. Now that it is appropriately named and commented with what the values are, I think it's more clear what the data represents.

Reply:
It's great that there's a comment there now, and it's definitely "more clear", but it's still a long way from being as obviously readable as it could trivially could be as a structure.

Reply:
I've added a structure to represent these arguments.